package.json

```diff
@@ -52,11 +52,11 @@
     ]
   },
   "dependencies": {
+    "dotenv": "^16.3.1",
```

Comment:
why do we need dotenv here?

Reply:
For testing, we need to provide apiKey so in the test file it is given through env.

Reply:
I see. I think it should be fine if you just use hard coded key in the test file itself and then provide it to smart account config.

Reply:
Ok then wil remove dotenv and use hardcoded values